deps: github.com/oras-project/oras-credentials-go was merged into oras.land/oras-go/v2

pdecat opened the pull request:
https://github.com/oras-project/oras-go/tree/main/registry/remote/credentials is now part of a the [v2.4.0 release](https://github.com/oras-project/oras-go/releases/tag/v2.4.0).

pdecat commented:
This will have to wait for v2.4.0 of oras-go: https://github.com/oras-project/oras-go/milestone/17

pdecat commented:
keep

kaidaguerre commented:
Hey @pdecat sorry for the delay on this one - let's try to get it merged in. Could you rebase

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/turbot/steampipe?pullRequest=3900) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/turbot/steampipe?pullRequest=3900) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/turbot/steampipe?pullRequest=3900) it.</sub>

pdecat commented:
Rebased on main and updated to use oras.land/oras-go/v2 v2.3.1-0.20231015113306-e8e4f843df97

Still waiting on a proper release to mark the PR as ready.

Only two issues left open for the milestone :crossed_fingers:  https://github.com/oras-project/oras-go/milestone/17

pdecat commented:
https://github.com/oras-project/oras-go/releases/tag/v2.4.0 is out!

pdecat commented:
Looks like the failing check is caused by a network timeout:
```
Warning: Failed to download action 'https://api.github.com/repos/actions/download-artifact/tarball/9bc31d5ccc31df68ecc42ccf4149144866c47d8a'. Error: Operation timed out (codeload.github.com:443)
Warning: Back off 17.101 seconds before retry.
Error: Operation timed out (api.github.com:443)
```
https://github.com/turbot/steampipe/actions/runs/7710558276/job/21014172562?pr=3900

kaidaguerre commented:
Hey @pdecat could you rebase this one more time and I'll finally merge it.

kaidaguerre commented:
Finally! Thanks for your patience
